elixir/apps/web/lib/web/live/actors/users/new_identity.ex

```diff
@@ -130,4 +132,15 @@ defmodule Web.Actors.Users.NewIdentity do
       _ -> ~p"/#{socket.assigns.account}/actors/#{socket.assigns.actor}"
     end
   end
+
+  defp add_email(attrs) do
+    identifier = attrs["provider_identifier"]
+
+    # Check if the identifier is in the general shape of an email address
+    if identifier =~ ~r/^[^\s]+@[^\s]+\.[^\s]+$/ do
```

Comment:
I see this regex literal used in a few places. What do you think about making this a function on AuthIdentity or somewhere?

Reply:
👍 Yep, will update

Reply:
Have moved this to a couple of functions in `domain/auth.ex`